sdks/browser-sdk/CHANGELOG.md

```diff
@@ -1,5 +1,186 @@
 # @xmtp/browser-sdk
 
+## 2.0.0
+
+This release focuses on new features, stability, and performance.
+
+## Upgrade from 1.x to 2.0.0
+
+Use the information in these release notes to upgrade from `@xmtp/browser-sdk` 1.x to 2.0.0.
+
+> [!IMPORTANT]  
+> **Upgrading from a legacy XMTP V2 SDK?** Legacy XMTP V2 SDKs include JavaScript SDK vx.x.x. To learn how to upgrade to stable XMTP V3, be sure to also see important dates and considerations in [Upgrade from a legacy XMTP V2 SDK](https://docs.xmtp.org/upgrade-from-legacy-V2).
```

Comment:
I'd like to suggest deleting this callout because the concern described here is already covered as a part of the upgrade path from legacy V2 and legacy V3 - culminating in the dev upgrading to v1.1.4.

Once the upgrade to v1.1.4 is complete - the path is isolated to v1.1.4 to v2, with the breaking changes described in these notes.

```suggestion
```